### ISSUE TITLE
bug: (YouTube - Seekbar thumbnail): Showing a different preview

### ISSUE THREAD

**Issue by Magdy44** [ReVanced/revanced-patches#3370]
### Bug description

Sometimes when i am using the seekbar thumbnail it shows the preview of a completely different video and when that happens i have to close the app every time to work properly. I tried the old seekbar thumbnail patch and it also didn't work.

Is there a fix for this?

![1701475975120](https://github.com/ReVanced/revanced-patches/assets/152651678/69308673-0429-4c3b-99a6-710422fe732d)

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Magdy44** [comment]
> Are you using the latest patches?

Yes, i am using (v2.202.0) patch with the recommended version (18.45.43)

**Comment by Magdy44** [comment]
> Do you have play 'playback in feed' turned on? (In YouTube general settings) If so, try turning it off and see if that helps.
> 
> Try to find reproduction steps that recreate the issue. Otherwise it's very difficult to fix if the problem cannot be reliably recreated.

I already had it off, but i turned it on to see if it helps and it didn't.

I tried different things before like:
- turning off spoof device dimension
- turning off spoof signature in feed
- Disable precious seeking gesture
- uninstalling and reinstalling revanced
- installing revanced extended

And they also didn't help.

**Comment by oSumAtrIX** [comment]
Please do this

> Try to find reproduction steps that recreate the issue. Otherwise it's very difficult to fix if the problem cannot be reliably recreated.

**Comment by Pamilg8** [comment]
As you can see in the image, Restore old seekbar thumbnails" not applied.🤔

**Comment by Magdy44** [comment]
> As you can see in the image, Restore old seekbar thumbnails" not applied. I think YouTube is the problem.ReVanced makes the thumbnail very blurry when scrolling in the search bar. Better quality on YouTube using the same fast Wi-Fi.I'm not talking about the Legacy miniature, but the current

Is it actually a YouTube problem? Yeah the thumbnails are better quality but i already checked if YouTube had that bug and it didn't.

I also used the old seekbar thumbnails patch and it didn't help.

**Comment by Pamilg8** [comment]
ReVanced has a blurring problem. And yes, your problem may be a YouTube problem. I think I encountered it
